web/Controllers/AbstractController.cs

```diff
@@ -11,4 +11,27 @@ protected BricksAndHeartsUser GetCurrentUser()
 
         return (BricksAndHeartsUser)User.Identity;
     }
+    
+    protected void FlashRequestSuccess(ILogger logger, BricksAndHeartsUser user, string requestType)

```

Comment:
I think our input isn't much of a type, maybe `requestAction` would be a better name? 🙂